firebase-installations/src/test/java/com/google/firebase/installations/FirebaseInstallationsRegistrarTest.java

```diff
@@ -14,5 +14,42 @@
 
 package com.google.firebase.installations;
 
+import static org.junit.Assert.assertNotNull;
+
+import androidx.test.core.app.ApplicationProvider;
+import com.google.firebase.FirebaseApp;
+import com.google.firebase.FirebaseOptions;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+
 /** Tests for {@link FirebaseInstallationsRegistrar}. */
-public class FirebaseInstallationsRegistrarTest {}
+@RunWith(RobolectricTestRunner.class)
+public class FirebaseInstallationsRegistrarTest {
+  @Before
+  public void setUp() {
+    FirebaseApp.clearInstancesForTest();
+  }
+
+  @Test
+  public void getFirebaseInstallationsInstance() {
+    FirebaseApp defaultApp =
+        FirebaseApp.initializeApp(
+            ApplicationProvider.getApplicationContext(),
+            new FirebaseOptions.Builder().setApplicationId("1:123456789:android:abcdef").build());
+
+    FirebaseApp anotherApp =
+        FirebaseApp.initializeApp(
+            ApplicationProvider.getApplicationContext(),
+            new FirebaseOptions.Builder().setApplicationId("1:987654321:android:abcdef").build(),
+            "firebase_app_1");
```

Comment:
this seems to cover 2 tests in once:
i.e. getting a different instance for a different App-ID, but also getting a different instance for different (nick)name.
Maybe those can be distinguished into 2 separate tests, i.e getting a different instance for different App-ID, and one getting different instance for different (nick)name?

Reply:
We are not able to create a diff firebase app with different App-ID without a nick-name. As a result, they cant be split in two. On a high level test the instance creation.